packages/clerk-js/bundlewatch.config.json

```diff
@@ -2,8 +2,8 @@
   "files": [
     { "path": "./dist/clerk.js", "maxSize": "560kB" },
     { "path": "./dist/clerk.browser.js", "maxSize": "75kB" },
-    { "path": "./dist/clerk.headless.js", "maxSize": "48.3KB" },
-    { "path": "./dist/ui-common*.js", "maxSize": "89KB" },
+    { "path": "./dist/clerk.headless.js", "maxSize": "48.7KB" },
```

Comment:
❓ How much margin does this leave in the bundle size?

Reply:
I think we should increase the margin here to at least 50KB - the reason why, I've started this branch on top of another to continue implementing the feature, and added a couple of lines on `Clerk` core, which increased to the following:

![CleanShot 2025-03-11 at 14 52 06](https://github.com/user-attachments/assets/a9d28914-70f1-4779-8d51-ec83ef430c31)